bedrock/firefox/templates/firefox/firstrun/base.html

```diff
@@ -6,8 +6,27 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% if l10n_has_tag('firstrun_meta_info') %}
```

Comment:
Ok, for some reason (that I've not quite worked out yet), this l10n tag does not work here anyway. I'm still not quite clear on why we need it though, seeing as the new /firstrun template is the only translated template inheriting from it. Perhaps @flodolo can help explain the reasoning here for why we need it?